packages/sdk/src/utils/misc-utils.ts

```diff
@@ -0,0 +1,9 @@
+// TODO: A lot of this stuff could probably live in core-utils instead.
+// Review this file eventually for stuff that could go into core-utils.
+export const omit = (obj: any, ...keys: string[]) => {
+  return keys.reduce((acc, key) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { [key]: omitted, ...rest } = acc
```

Comment:
I'm confused here - is `omitted` defined? Also I think this could be implemented more cleanly by copying `obj` and then iterating over keys and deleting each one using `delete`

Reply:
Yeah I guess this is personal preference. Either approach works here (`delete` or destructuring). `omitted` is undefined and just used to pull the variable out.

Reply:
I'll edit this to use `delete`, I think that's more straightforward

Reply:
Replaced to use `delete`. Resolving this comment.